docs/modules/ROOT/pages/index.adoc

```diff
@@ -498,21 +508,186 @@ In each of the above sections, the following changes MUST be applied:
+
+Web browsers provide a well-known example of transitive trust. Browsers have direct relationships with relatively few root trust anchors. Those anchors, known as _root certificate authorities,_ in turn issue credentials to _certificate authorities_ who then issue credentials to individual web site operators who then sign the content presented to a browser. This pattern may be repeated with multiple layers of intermediate certificate authorities. The web browser evaluates this entire chain of credentials when deciding whether to present a web site as trusted or not.
+
+=== Trust scenarios in identity assertion
+
+The `signature` in an *<<_identity_assertion,identity assertion>>* could be considered as a new credential documenting the relationship between _<<_named_actor,named actor>>_ and _<<C2PA asset>>,_ which will be referred to in this section as an _asset-specific credential._
+
+IMPORTANT: “Content Credential” is a trademarked term that refers to _<<_c2pa_manifest,C2PA Manifests>>_ and MUST NOT be used in reference to *<<_identity_assertion,identity assertion>>* signatures.
+
+An _<<_identity_asertion_consumer,identity assertion consumer>>_ SHOULD maintain a list of trust relationships that it is prepared to accept when interpreting any *<<_identity_assertion,identity assertion>>.*
+
+There are several possible relationships between the implementation of the *<<_identity_assertion,identity assertion>>,* _<<_named_actor,named actor>>,_ and _credential issuer,_ as documented in the following subsections.
+
+IMPORTANT: The trust decisions described in the scenarios should only be evaluated once the *<<_identity_assertion,identity assertion>>* and the signature material within have been successfully validated as described in xref:_validating_the_identity_assertion[xrefstyle=full].
+
+==== Named actor as issuer
+
+In this scenario, the _<<_credential_holder,credential holder>>_ possesses a credential that describes the _<<_named_actor,named actor>>_ and is provisioned with the ability to generate digital signatures on the _<<_named_actor,named actor’s>>_ behalf.
+
+This scenario is implicit in the X.509 certificate-based workflow as described in xref:_x_509_certificates_and_cose_signatures[xrefstyle=full]. Other credential types MAY also follow this scenario.
+
+The _<<_credential_holder,credential holder>>_ uses this signature authority directly to generate the _asset-specific credential,_ as depicted in the following diagram, which is non-normative:
+
+.Named actor as issuer
+image::trust-model/named-actor-as-issuer.drawio.svg[Named actor as issuer,width=570,height=240,align="center"]
+
+In this scenario, the _<<_identity_assertion_consumer,identity assertion consumer>>_ SHOULD make its trust decision based on the following predicates:
+
+. Is there a direct trust relationship with the _<<_named_actor,named actor>>?_ If so, the *<<_identity_assertion,identity assertion>>* SHOULD be treated as *trusted.*
```

Comment:
@scouten-adobe direct trust relationship is a rare case; perhaps adjust the line weight of that line to make that more obvious?